dask/dataframe/core.py

```diff
@@ -1090,6 +1101,55 @@ def _aca_agg(self, token, func, aggfunc=None, **kwargs):
                    columns=return_scalar, token=self._token_prefix + token,
                    **kwargs)
 
+    def apply_concat_apply(self, func, aggfunc=None, func_kwargs=None,
+                           aggfunc_kwargs=None):
+        """ Apply a function to each partition, the concat, then apply again
+
+        Useful to perform easily parallelizable aggregations based on data from
+        the whole series, like minimum, maximum, mean, etc.
+
+        Parameters
+        ----------
+
+        func: function [partition-per-arg] -> partition
+            Function to operate on each partition
+        aggfunc: function concatenated-partition -> partition
+            Function to operate on the concatenated result of func. Defaults to
```

Comment:
I wonder if some description of the behavior of  `_concat` isn't in order. (As a recent user of `_aca_agg`, I was a bit surprised when arrays were allowed but the concatenation was not `numpy.concatenate`.)